app/src/main/java/kr/co/marubook/presentation/search/SearchViewModel.kt

```diff
@@ -1,11 +1,35 @@
 package kr.co.marubook.presentation.search
 
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
 import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import kr.co.marubook.domain.entity.SearchItem
+import kr.co.marubook.domain.repository.SearchRepository
 import javax.inject.Inject
-import kr.co.marubook.data.remote.repository.SearchRepository
 
 @HiltViewModel
 class SearchViewModel @Inject constructor(
     private val searchRepository: SearchRepository
-) : ViewModel()
+) : ViewModel() {
+    private val _bookList = MutableLiveData<List<SearchItem.BookItem>>()
+    val bookList: LiveData<List<SearchItem.BookItem>>
+        get() = _bookList
+    private val _moimList = MutableLiveData<List<SearchItem.MoimItem>>()
+    val moimList: LiveData<List<SearchItem.MoimItem>>
+        get() = _moimList
+
+    fun getBookList() {
+        viewModelScope.launch {
+            _bookList.value = searchRepository.getBookList()
+        }
```

Comment:
runCatching 이용해서 error handling 하시는걸 추천드립니다.

```suggestion
        viewModelScope.launch {
            runCatching { searchRepository.getBookList() }
                .onSuccess { _bookList.value = it }
                .onFailure { it.printStackTrace() }
        }
```

Reply:
오 감사합니다 runCatching에 대해 잘모르는데 좀 더 공부해볼게요 ! (간단해보이지만)